src/actinia_core/cli/webhook_server.py

```diff
@@ -61,6 +64,8 @@ def update():
         pass
     except TypeError:
```

Comment:
Not quite sure, but to me `TypeError` sounds more general than `UnsupportedMediaType`, so I would have expected the order of the two to be the other way around.

Reply:
I changed it.